transformation/src/main/scala/org/broadinstitute/monster/clinvar/parsers/ParsedArchive.scala

```diff
@@ -82,7 +82,12 @@ object ParsedArchive {
 
     // Since IncludedRecords don't contain meaningful provenance, we only
```

Comment:
Might be a dumb question - do we still want to collect IncludedRecords in the block above (in variationRecord) even though we won't process them? Or is it that we just don't need the provenance data ("@VariationID", "@VariationName", "@VariationType") from those particular records?

Reply:
Not a dumb question. The variation data in "included" records is important because it's the only place we can find information about "child" variations in certain situations. Everything but the variation/gene data in those records is fake, so we don't even bother with further processing.